lms/djangoapps/labster_course_license/models.py

```diff
@@ -37,6 +38,40 @@ def set_license(cls, course_id, license_code):
         except cls.DoesNotExist:
             course_license = cls.objects.create(course_id=course_id, license_code=license_code)
         course_license.save()
+        return course_license
 
     def __unicode__(self):
         return unicode(repr(self))
+
+
+class LicensedCoursewareItems(models.Model):
+    """
+    Stores all licensed simulations within block including ones from child blocks.
+    """
+    course_license = models.ForeignKey(CourseLicense)
+    block = models.CharField(max_length=64)
+    licensed_simulations = models.TextField()
+
+
+class LicensedSimulations(models.Model):
+    """
+    Keeps cached info from Labster API about licensed simulations for license.
+    """
+    course_license = models.ForeignKey(CourseLicense)
+    licensed_simulations = models.TextField(blank=True, null=True)
```

Comment:
Doesn't it looks like duplication of data between models `LicensedSimulations` and `LicensedCoursewareItems`?